bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/storage/ldb/DbLedgerStorage.java

```diff
@@ -359,4 +360,39 @@ public static void readLedgerIndexEntries(long ledgerId, ServerConfiguration ser
     public void forceGC() {
         ledgerStorageList.stream().forEach(SingleDirectoryDbLedgerStorage::forceGC);
     }
+
+    @Override
+    public boolean isInForceGC() {
+        return ledgerStorageList.stream().anyMatch(SingleDirectoryDbLedgerStorage::isInForceGC);
+    }
+
+    @Override
+    public GarbageCollectionStatus getGarbageCollectionStatus() {
```

Comment:
I suggest returning `List<GarbageCollectionStatus>`, because in DbLedgerStorage is a list of storage instances.